obstacle_stop_planner_refine/package.xml

```diff
@@ -8,7 +8,7 @@
 
   <license>Apache 2.0</license>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
```

Comment:
[Question] Why did you change this? Some errors have occurred?

Reply:
I got error when I use `ament_cmake`. Maybe `rosdep` needs `ament_cmake_auto` to find package.
In other environment, `ament_cmake_auto` has already installed so that there are no errors. But I think using `ament_cmake_auto` is better to avoid errors.

https://github.com/tier4/obstacle_stop_planner_refine/runs/2021744768?check_suite_focus=true

```
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
  Starting function 'build_target_workspace'
  Starting >>> obstacle_stop_planner_refine
  --- stderr: obstacle_stop_planner_refine
  CMake Error at CMakeLists.txt:14 (find_package):
    By not providing "Findament_cmake_auto.cmake" in CMAKE_MODULE_PATH this
    project has asked CMake to find a package configuration file provided by
    "ament_cmake_auto", but CMake did not find one.
  
    Could not find a package configuration file provided by "ament_cmake_auto"
    with any of the following names:
  
      ament_cmake_autoConfig.cmake
      ament_cmake_auto-config.cmake
  
    Add the installation prefix of "ament_cmake_auto" to CMAKE_PREFIX_PATH or
    set "ament_cmake_auto_DIR" to a directory containing one of the above
    files.  If "ament_cmake_auto" provides a separate development package or
    SDK, be sure it has been installed.
  
  
  ---
```